assets/src/styles/components/_well-log.scss

```diff
@@ -24,11 +24,20 @@ table {
     text-align: left;
 }
 
+tbody {
+    padding: 12px;
+}
+
 #all:checked ~ table .screen,
 #all:checked ~ table .casing,
 #screens:checked ~ table .screen,
 #casings:checked ~ table .casing {
     display: table-row;
+    /* add breakpoint to allow Well Construction table to be width: 100%
+    when table is displayed for narrow view ports. */
+    @media only screen and (max-width: 600px) {
```

Comment:
Would it make sense here to use one of the USWDS breakpoints here, like `tablet`?

Reply:
Yup. Done.